generators/profilegen/src/profile_generator_options_parser.rb

```diff
@@ -13,6 +13,9 @@
 #    -h, --help : Displays help for the profile generator
 #    -d, --dir_metadata : File that contains metadata info about RPs
 #    -p, --profile : Input file contains the profile generation info about RPs, versions, etc.
+#    -m, --mode : Mode can be 'rollup'/ 'management'
```

Comment:
would we not support "dataplane"? what would be different profile generator wise? is there a reason why the options are not rolloup & individual as in other cases?